jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -193,4 +206,19 @@ static String joinAsJsonArray(List<String> items) {
 
     return resultString.toString();
   }
+
+  /**
+   * Builds a list of Dockerfile "EXPOSE" items.
+   *
+   * @param exposedPorts the list of ports numbers/ranges to expose
+   * @return a string containing an EXPOSE command for each of the entries
+   */
+  @VisibleForTesting
+  static String makeExposeItems(List<String> exposedPorts) {
+    StringBuilder resultString = new StringBuilder();
+    for (String port : exposedPorts) {
+      resultString.append("EXPOSE ").append(port).append("\n");
```

Comment:
nit: can be '\n'

Reply:
Also, if you don't want the extra new line at the end of the list, you can do a join instead.